WMass/python/analyzers/ntupleTypes.py

```diff
@@ -12,35 +12,35 @@
-    NTupleVariable("hcalOverEcal", lambda x: x.full5x5_hcalOverEcal() if (abs(x.pdgId())==11 and hasattr(x,"full5x5_hcalOverEcal")) else -999, help="Electron full5x5_hcalOverEcal"),
-    NTupleVariable("eSuperClusterOverP", lambda x: x.eSuperClusterOverP() if (abs(x.pdgId())==11 and hasattr(x,"eSuperClusterOverP")) else -999, help="Electron eSuperClusterOverP"),
-    NTupleVariable("matchedTrgObjElePt" , lambda x: x.matchedTrgObjwmassEle.pt() if  x.matchedTrgObjwmassEle else -999.        , help="Matched trigger object (cone dR<0.3) pT to Ele27_WPTight"),
-    NTupleVariable("matchedTrgObjEleDR" , lambda x: deltaR(x, x.matchedTrgObjwmassEle) if  x.matchedTrgObjwmassEle else -999.  , help="Matched trigger object (cone dR<0.3) dR to Ele27_WPTight"),
+    #NTupleVariable("ecalPFClusterIso", lambda lepton :  lepton.ecalPFClusterIso() if abs(lepton.pdgId())==11 else -999, help="Electron ecalPFClusterIso"),
+    #NTupleVariable("hcalPFClusterIso", lambda lepton :  lepton.hcalPFClusterIso() if abs(lepton.pdgId())==11 else -999, help="Electron hcalPFClusterIso"),
+    #NTupleVariable("dr03TkSumPt", lambda lepton: lepton.dr03TkSumPt() if abs(lepton.pdgId())==11 else -999, help="Electron dr03TkSumPt isolation"),
+    #NTupleVariable("trackIso", lambda lepton :  lepton.trackIso() if abs(lepton.pdgId())==11 else -999, help="Electron trackIso (in cone of 0.4)"),
+    #NTupleVariable("trackIso03", lambda lepton :  lepton.isolationR03().sumPt/lepton.pt() if abs(lepton.pdgId())==13 else -999, help="Muon track isolation in cone 0.3"),
+    #NTupleVariable("etaSc", lambda x : x.superCluster().eta() if abs(x.pdgId())==11 else -100, help="Electron supercluster pseudorapidity"),
+    #NTupleVariable("energySc", lambda x : x.superCluster().energy() if abs(x.pdgId())==11 else -100, help="Electron supercluster pseudorapidity"),
+    #NTupleVariable("e5x5", lambda x: x.full5x5_e5x5() if (abs(x.pdgId())==11 and hasattr(x,"full5x5_e5x5")) else -999, help="Electron full5x5_e5x5"),
+    #NTupleVariable("sigmaIetaIeta", lambda x: x.full5x5_sigmaIetaIeta() if (abs(x.pdgId())==11 and hasattr(x,"full5x5_sigmaIetaIeta")) else -999, help="Electron full5x5_sigmaIetaIeta"),
+    #NTupleVariable("hcalOverEcal", lambda x: x.full5x5_hcalOverEcal() if (abs(x.pdgId())==11 and hasattr(x,"full5x5_hcalOverEcal")) else -999, help="Electron full5x5_hcalOverEcal"),
+    #NTupleVariable("eSuperClusterOverP", lambda x: x.eSuperClusterOverP() if (abs(x.pdgId())==11 and hasattr(x,"eSuperClusterOverP")) else -999, help="Electron eSuperClusterOverP"),
+    #NTupleVariable("matchedTrgObjElePt" , lambda x: x.matchedTrgObjwmassEle.pt() if  x.matchedTrgObjwmassEle else -999.        , help="Matched trigger object (cone dR<0.3) pT to Ele27_WPTight"),
+    #NTupleVariable("matchedTrgObjEleDR" , lambda x: deltaR(x, x.matchedTrgObjwmassEle) if  x.matchedTrgObjwmassEle else -999.  , help="Matched trigger object (cone dR<0.3) dR to Ele27_WPTight"),
```

Comment:
Not that I care much, but why removing all the electron ID variables? Even if running muons only, the electrons in the Z->mumu MC and SingleMuon in data will be small, and won't add much space. Still may be useful for vetoing extra leptons.

Reply:
It was just to remove a bunch of branches I was not using (and many variables were never used even with electrons), to maintain the size of the trees as small as possible

I can add them back